besu/src/main/java/org/hyperledger/besu/cli/options/PrunerOptions.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.cli.options;
+
+import org.hyperledger.besu.ethereum.worldstate.PrunerConfiguration;
+
+import java.util.Arrays;
+import java.util.List;
+
+import picocli.CommandLine;
+
+public class PrunerOptions implements CLIOptions<PrunerConfiguration> {
+  private static final String BLOCKS_RETAINED_FLAG = "--Xpruning-blocks-retained";
+  private static final String BLOCK_CONFIRMATIONS_FLAG = "--Xpruning-block-confirmations";
+
+  @CommandLine.Option(
+      names = {BLOCKS_RETAINED_FLAG},
+      hidden = true,
+      defaultValue = "1024",
+      paramLabel = "<INTEGER>",
+      description =
+          "Minimum number of recent blocks for which to keep entire world state (default: ${DEFAULT-VALUE})",
```

Comment:
In the CLI message defined before, you used the term `retention period`. Should we be consistent here?

Reply:
I don't see a difference between this line and the previous line for the same option

Reply:
I meant the one in `BesuCommand` :)

`Enable pruning of world state of blocks older than the retention period (default: true if fast sync is enabled, false otherwise)`

And here:

`Minimum number of recent blocks for which to keep entire world state`

Reply:
Is `retention period` the same as `Minimum number of recent blocks for which to keep entire world state`?

Reply:
Ah, ok. It is the same but I think I want to remove the retention period call out for the reasons mentioned in the other comment.